docs/manual/java/releases/Migration15.md

```diff
@@ -46,30 +46,78 @@ were never intended for public consumption, and therefore have been marked depre
 
-[Lighbend Orchestration](https://developer.lightbend.com/docs/lightbend-orchestration/current/) is an open-source suite of tools that helps you deploy Lagom services to Kubernetes and DC/OS. It provides an easy way to create Docker images for your applications and introduces an automated process for generating Kubernetes and DC/OS resource and configuration files for you from those images. This process helps reduce the friction between development and operations. If you are using Kubernetes or DC/OS, or interested in trying one of these platforms, we encourage you to read the Lightbend Orchestration documentation to understand how to use it with Lagom and other components of the [Lightbend Reactive Platform](https://www.lightbend.com/products/reactive-platform).
+See Deployment below.
 
-If you are not using Kubernetes or DC/OS, you must configure your services in a way that suits your production environment. See [[Running Lagom in production|ProductionOverview]] for more information.
+## Lightbend Orchestration
 
-If you are using ConductR in production and need assistance migrating to other solutions please [contact Lightbend](https://www.lightbend.com/contact).
+[Lightbend Orchestration](https://developer.lightbend.com/docs/lightbend-orchestration/current/) is not supported with Lagom 1.5. If you used Lightbend Orchestration for your deployment you will have to migrate to a manual process. This migration may happen on the same commit where you upgrade to Lagom 1.5.0 or you can upgrade your deployment process on a first step (still using Lagom 1.4.x) and later upgrade to Lagom 1.5.0.
 
-## TLS Support
+See Deployment below.
 
-## Lightbend Orchestration
+## Deployment
 
-TODO (new version required)
+ConductR tooling and Lightbend Orchestration handled all the required pieces to deploy on ConductR and Kubernetes or DC/OS. Lagom 1.5.0 only supports a manually maintained deployment process.
 
-## Cluster Formation
+In particular, ConductR tooling and Lightbend Orchestration handled some or all of the following:
 
-A new mechanism to form and [[join an Akka Cluster|Cluster#Joining]] is introduced in Lagom 1.5. Apart from the original `Manual Cluster Formation` the new `Akka Cluster Bootstrap` is now supported. This new mechanism is introduced with lower precedence than `Manual Cluster Formation` so if you rely on the use of a list of `seed-nodes` then everything will work as before. On the other hand, `Akka Cluster Bootstrap` takes precedence over the `join-self` cluster formation for single-node clusters. If you use single-node clusters via `join-self` you will have to explicitly disable `Akka Cluster Bootstrap`:
+1. extend the your application with: cluster bootstrapping, akka management and health checks
+2. Service Location
+3. setup and produce docker images
+4. prepare the deployment specs for the target orchestrator
```

Comment:
s/prepare/preparing/

You start the sentence with "ConductR and LBO handled some or all of the following" so you have to continue using the past tense for all the elements ("setting up and producing docker images", "extending ..." etc)

Reply:
So instead of:

```
In particular, ConductR tooling and Lightbend Orchestration handled some or all of the following:

1. extend the application with: cluster bootstrapping, akka management and health checks
2. Service Location
3. setup and produce docker images
4. prepare the deployment specs for the target orchestrator
5. Secrets
```

I should use:

```
In particular, ConductR tooling and Lightbend Orchestration handled some or all of the following:

1. extending the application with: cluster bootstrapping, akka management and health checks
2. providing a Service Location implementation
3. setting up and producing docker images
4. preparing the deployment specs for the target orchestrator
5. accessing Secrets
```
?

Reply:
Yes, that's fine.  You can also keep "Service Location" and "Secrets" as nouns.